### PR TITLE
DSPDC-521 Capitalize SQL keywords.

### PR DESCRIPTION
**Pull request by danxmoran**
TODO: Find the TODO about this, and remove it

### CODE REVIEW

File: manager/src/test/scala/org/broadinstitute/transporter/transfer/TransferListenerSpec.scala [hunk 1]
```diff
@@ -315,10 +315,10 @@ class TransferListenerSpec extends PostgresSpec with MockFactory with EitherValu
 
     for {
       _ <- ids.traverse_ { id =>
-        sql"update transfers set status = 'submitted' where id = $id".update.run
+        sql"UPDATE transfers SET status = 'submitted' WHERE id = $id".update.run
       }.transact(tx)
       _ <- List(doUpdate, doRecord, doRecord, doUpdate, doRecord, doUpdate).parSequence_
-      statuses <- sql"select count(1) from transfers where status = 'succeeded'"
+      statuses <- sql"SELECT COUNT(1) FROM transfers WHERE status = 'succeeded'"
```
Comment (raaidbroad) [inline manager/src/test/scala/org/broadinstitute/transporter/transfer/TransferListenerSpec.scala]: unrelated to this refactor, but: I think the 'succeeded' and any other places where the status might be written out as the literal here should instead refer to the right `TransferStatus`

**Comment by danxmoran** [comment]
@benjamincarlin I'm going to merge since this is high-risk for merge conflicts, but still feel free to comment if you see problems and I'll open a separate PR to fix!

File: manager/src/main/scala/org/broadinstitute/transporter/transfer/TransferController.scala [hunk 2]
```diff
@@ -49,7 +47,7 @@ class TransferController(
   /** Get the total number of transfer requests stored by Transporter. */
   def countRequests: IO[Long] =
     Fragment
-      .const(s"select count(1) from $RequestsTable")
+      .const(s"SELECT COUNT(1) from $RequestsTable")
```
Comment (benjamincarlin) [inline manager/src/main/scala/org/broadinstitute/transporter/transfer/TransferController.scala]: `FROM`?

Comment (danxmoran) [reply manager/src/main/scala/org/broadinstitute/transporter/transfer/TransferController.scala]: !!! crap, good catch

Comment (danxmoran) [reply manager/src/main/scala/org/broadinstitute/transporter/transfer/TransferController.scala]: Pushed the change to master: https://github.com/DataBiosphere/transporter/commit/e12afc34d7082d62d3c1309f6b45a44bf6d17ca9

**Pull request merged**